src/docs/Contributing.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contributing"/>
+
+# Contribution Guidelines
+
+moja global welcomes contributions to the community website as well as the UI-library. If you want to contribute a new component to the library, please follow the steps given below:
```

Comment:
The above one got tagged by mistake. Please refer these lines.

Reply:
Yes, I'll work on it!